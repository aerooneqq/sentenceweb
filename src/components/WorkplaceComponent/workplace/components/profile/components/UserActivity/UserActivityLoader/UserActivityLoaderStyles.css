.userActivityLoaderContainer { 
    margin-left: 10px;

    height: 180px;

    display: flex;
    flex-direction: row;
}

.userActivityAtomLoaderCont { 
    margin-left: 10px;
    margin-top: 10px;

    height: 180px;
    width: 170px; 

    display: flex; 
    flex-direction: column;
}

.userActivityAtomLoaderMonthName { 
    width: 70px; 
    height: 20px; 

    background: #c0c0c0;

    border-radius: 5px;
}

.userActivityAtomLoaderMonth { 
    width: 100%;
    height: 140px; 

    margin-top: 10px; 

    background: #e0e0e0;

    border-radius: 5px;
}