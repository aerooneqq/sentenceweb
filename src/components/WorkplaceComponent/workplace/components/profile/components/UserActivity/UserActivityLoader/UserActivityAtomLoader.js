import React, {Component} from "react";

import "./UserActivityLoaderStyles.css";

export default class UserActivityAtomLoader extends Component { 

    constructor(props) { 
        super(props);
    }

    render() { 
        return ( 
            <div className = "userActivityAtomLoaderCont">
                
                <div className = "userActivityAtomLoaderMonthName" />
                <div className = "userActivityAtomLoaderMonth" />

            </div>
        )
    }
}