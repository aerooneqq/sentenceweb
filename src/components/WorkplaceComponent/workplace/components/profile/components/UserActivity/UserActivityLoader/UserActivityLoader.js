import React, {Component} from "react";

//Styles
import "./UserActivityLoaderStyles.css";

import UserActivityAtomLoader from "./UserActivityAtomLoader";

export default class UserActivityLoader extends Component { 

    constructor(props) { 
        super(props);
    }

    render() { 
        return (
            <div className = "userActivityLoaderContainer">
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
                <UserActivityAtomLoader />
            </div>
        )
    }

}