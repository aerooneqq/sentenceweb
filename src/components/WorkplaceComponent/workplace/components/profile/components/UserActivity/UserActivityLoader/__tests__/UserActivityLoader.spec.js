import React from "react";
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json"; 
import UserActivityLoader from "../UserActivityLoader";
import UserActivityAtomLoader from "../UserActivityAtomLoader";

describe("UserActivityLoader", () => { 
    it("should render correctly", () => { 
        const userActivityLoader = shallow(<UserActivityLoader />);

        expect(shallowToJson(userActivityLoader)).toMatchSnapshot();
    });

    it("should have 12 UserActivityAtomLoaders", () => { 

        expect(shallow(<UserActivityLoader />).find(UserActivityAtomLoader).length).toBe(12);
    });
});