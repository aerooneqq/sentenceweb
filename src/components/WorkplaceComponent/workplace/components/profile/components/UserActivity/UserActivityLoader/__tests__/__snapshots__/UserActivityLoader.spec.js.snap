// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserActivityLoader should render correctly 1`] = `
<div
  className="userActivityLoaderContainer"
>
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
  <UserActivityAtomLoader />
</div>
`;
